docs/general_usage.md

```diff
@@ -33,6 +33,39 @@ The CLI will attempt to load credentials in the following order of priority:
 1. From environment variables, if set
 1. From the active profile
 
+## Subaccounts
+
+Instructions on how to create a Twilio subaccount can be found in the [Twilio docs](https://support.twilio.com/hc/en-us/articles/360011348693-View-and-Create-New-Twilio-Subaccounts).
+
+Once you have your subaccount SID, you can add the `--account-sid` parameter to your call to run a command on behalf of a subaccount, provided it's a `core` command. For all other commands, please use profiles (via `twilio login` to create a new profile or `twilio profiles` to manage existing credentials).
```

Comment:
Not really _on behalf of_, that sounds more like impersonation. Maybe _to run a command on a specific subaccount_.

_Provided it's a `twilio api:core:` command_.

_please create a profile for the subaccount (e.g., `twilio login <subaccount SID>`)_